Dockerfile

```diff
@@ -119,6 +87,7 @@ COPY src/scripts /engine/src/scripts
 ## Default ENVs that can be overwritten
 ARG IASQL_ENV=local
 ENV IASQL_ENV=$IASQL_ENV
+ENV NEXT_PUBLIC_IASQL_ENV=$IASQL_ENV
```

Comment:
didn't we do this already in line 29? why repeat it?

Reply:
I think Alejandro added the one in line 29 back. This is to make the environment available at actual runtime. I'm not sure if the one he added actually does anything because we don't go through a `build` step now.

Reply:
and do we need this variable at build time or not anymore?